aiohttp/payload.py

```diff
@@ -146,6 +146,14 @@ def headers(self) -> Optional[_CIMultiDict]:
         """Custom item headers"""
         return self._headers
 
+    @property
+    def _binary_headers(self) -> Optional[bytes]:
```

Comment:
I'm really not sure about this property, that's why it's private, but that serialization happens in two different places. Any better ideas how to handle it?

Reply:
No idea, I don't recall this part of code